### PR TITLE
Product Details Page

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Pages
- Product Details page

Features 
- Brand and Category routing from home page
- Product details fetch and display
- Invalid route handling
- Sorting based on discounted price
- Error handling

Deployed version - https://weebofigurines.netlify.app/

### CODE REVIEW

File: src/context/providers/ProductProvider.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const ProductProvider = ({ children }) => {
     filterData
   )(state, state?.products || []);
 
-  console.log(filteredProducts, state.productsFilter);
+  // console.log(filteredProducts, state.productsFilter);
```
Comment (AmishMishra11) [inline src/context/providers/ProductProvider.js:39]: Remove comments

**Review by AmishMishra11 — APPROVED** [review]
You have followed good coding conventions
Good folder structure

**Review by ishapatell1 — APPROVED** [review]

File: src/context/providers/ProductProvider.js (hunk 1 above)
Comment (SukanyaSen530) [reply src/context/providers/ProductProvider.js:39]: Thanks for pointing out :) I will remove it in next PR.

**Review by ishapatell1 — COMMENTED** [review]
Hey sukanya, try adding screenshots next time!

**Pull request merged**